contracts/email-contracts/src/main/java/org/java/mentorship/contracts/email/client/EmailFeignClient.java

```diff
@@ -0,0 +1,31 @@
+package org.java.mentorship.contracts.email.client;
+
+import org.java.mentorship.contracts.email.dto.EmailRequest;
+import org.java.mentorship.contracts.email.dto.EmailResponse;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@FeignClient(value = "emailClient",
+        url = "${service.email.url}")
+public interface EmailFeignClient {
+
+    @RequestMapping(
+            method = RequestMethod.POST,
+            value = "/test"
+    )
+    EmailResponse sendTest(@RequestBody EmailRequest emailRequest);
+
+    @RequestMapping(
+            method = RequestMethod.POST,
+            value = "/verify"
+    )
+    EmailResponse sendVerification(@RequestBody EmailRequest emailRequest);
```

Comment:
This endpoint should be POST @ /emails
And the type of email should be present in EmailRequest